jelly/src/main/java/org/kohsuke/stapler/jelly/DefaultScriptInvoker.java

```diff
@@ -65,11 +66,15 @@ public void invokeScript(StaplerRequest req, StaplerResponse rsp, Script script,
 
     protected XMLOutput createXMLOutput(StaplerRequest req, StaplerResponse rsp, Script script, Object it) throws IOException {
         // TODO: make XMLOutput auto-close OutputStream to avoid leak
-        HTMLWriterOutput hwo = HTMLWriterOutput.create(createOutputStream(req, rsp, script, it));
         String ct = rsp.getContentType();
-        if (ct != null && !ct.startsWith("text/html"))
-            hwo.useHTML(false);
-        return hwo;
+        XMLOutput output;
```

Comment:
I'd be careful about merge conflicts here as this method was refactored in the security update.

Reply:
Hmmm ... looks like that hasn't been merged in yet.

Reply:
https://github.com/stapler/stapler/pull/166

Reply:
Looks like this method wasn't touched in the recent security update, but other unrelated methods in the file were.